packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/EditContract.jsx

```diff
@@ -0,0 +1,279 @@
+import { set } from 'lodash'
+import React, { useState } from 'react'
+import cx from 'classnames'
+
+import { useClient } from 'cozy-client'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
+import Button from 'cozy-ui/transpiled/react/Button'
+import { useI18n } from 'cozy-ui/transpiled/react'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+import Field from 'cozy-ui/transpiled/react/Field'
+import CollectionField from 'cozy-ui/transpiled/react/Labs/CollectionField'
+import Stack from 'cozy-ui/transpiled/react/Stack'
+import BaseContactPicker from 'cozy-ui/transpiled/react/ContactPicker'
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+import Modal from 'cozy-ui/transpiled/react/Modal'
+import { Media, Bd, Img } from 'cozy-ui/transpiled/react/Media'
+import NarrowContent from 'cozy-ui/transpiled/react/NarrowContent'
+import withLocales from '../../hoc/withLocales'
+import { SubTitle } from 'cozy-ui/transpiled/react/Text'
+
+import {
+  getAccountLabel,
+  getAccountInstitutionLabel,
+  getAccountType,
+  getAccountOwners
+} from './bankAccountHelpers'
+
+const ContactPicker = props => {
+  const { t } = useI18n()
+  // eslint-disable-next-line no-unused-vars
+  const { ...rest } = props
```

Comment:
why this spread?

Reply:
Ah, this is because previously, the picker would extract the t from the props, and would not forward it, it is no longer needed. Fixed.